docs/manual/guides/install-test-versions.de.md

```diff
@@ -0,0 +1,151 @@
+        "contao/manager-bundle": "4.9.*@RC",
+        "contao/news-bundle": "^4.9@RC",
+        "contao/newsletter-bundle": "^4.9@RC"
+    },
+    "conflict": {
+        "contao-components/installer": "<1.3"
+    },
+    "extra": {
+        "contao-component-dir": "assets",
+        "symfony": {
+            "require": "^4.2"
+        }
+    },
+    "scripts": {
+        "post-install-cmd": [
+            "Contao\\ManagerBundle\\Composer\\ScriptHandler::initializeApplication"
+        ],
+        "post-update-cmd": [
+            "Contao\\ManagerBundle\\Composer\\ScriptHandler::initializeApplication"
+        ]
+    }
+}
+```
+
+Beachte, dass dabei später auch die stabilen Versionen installiert 
+werden (wie z. B. `4.9.0`, `4.9.1` etc.), wenn eine Paketaktualisierung durchgeführt wird.
+
+
+## Entwicklerversionen installieren
+
+Während der Release Candidate und davor auch während der Entwicklungsphase einer
+Contao Version kann auch die Entwicklerversion zum Testen installiert werden. Auf
+diese Weise können die neuesten Änderungen sofort getestet werden, ohne auf einen
+neuen Release Candidate warten zu müssen. Natürlich kann dies auch instabilen Programmcode
+enthalten.
+
+Statt einer spezifischen _Version_ wird nun ein spezifischer _Branch_ aus dem öffentlichen
+Git Repository von Contao verlangt. Jede »Minor« Version von Contao hat ihren eigenen
+Entwicklungs-Branch, z. B. `4.9.x-dev` für Contao `4.9`. Der Branch für die neueste,
+sich gerade in Entwicklung befindliche, _zukünftige_ Version von Contao befindet 
+sich immer im `dev-master` Branch.
+
+Eine komplettes Beispiel einer `composer.json`, wo der Entwicklungs-Branch von
+Contao `4.9` installiert wird, sieht so aus:
+
+```json
+{
+    "name": "contao/managed-edition",
+    "type": "project",
+    "description": "Contao Open Source CMS",
+    "require": {
+        "php": "^7.1",
+        "contao/calendar-bundle": "4.9.x-dev",
+        "contao/comments-bundle": "4.9.x-dev",
+        "contao/conflicts": "@dev",
+        "contao/core-bundle": "4.9.x-dev",
+        "contao/faq-bundle": "4.9.x-dev",
+        "contao/installation-bundle": "4.9.x-dev",
+        "contao/listing-bundle": "4.9.x-dev",
+        "contao/manager-bundle": "4.9.x-dev",
+        "contao/news-bundle": "4.9.x-dev",
+        "contao/newsletter-bundle": "4.9.x-dev"
+    },
+    "conflict": {
+        "contao-components/installer": "<1.3"
+    },
+    "extra": {
+        "contao-component-dir": "assets",
+        "symfony": {
+            "require": "^4.2"
+        }
+    },
+    "scripts": {
+        "post-install-cmd": [
+            "Contao\\ManagerBundle\\Composer\\ScriptHandler::initializeApplication"
+        ],
+        "post-update-cmd": [
+            "Contao\\ManagerBundle\\Composer\\ScriptHandler::initializeApplication"
+        ]
+    }
+}
+```
+
+Jedes mal, wenn eine Paketaktualisierung durchgeführt wird, wird der neueste Programmcode
+aus diesem Branch des öffentlichen Git Repositorys von Contao geholt.
```

Comment:
«Repositories» anstelle von «Repositorys»

Reply:
Also ich habe gelernt, dass man in der Deutschen Rechtschreibung nicht die englischen Regeln der Plurarlbildung anwendet :). So schreibt man im Deutschen nicht "Hobbies" sondern "Hobbys" etc.

Reply:
Auch wenn ein deutsches Wort für [Repository](https://www.duden.de/rechtschreibung/Repositorium) vorhanden ist?

Reply:
Ah, und außerdem ist es ja kein Plural in diesem Fall 😁